web/modules/weather_blocks/src/Plugin/Block/WFOPromoBlock.php

```diff
@@ -25,17 +25,56 @@ public function build()
 
             $promo = $this->entityTypeService->getLatestNodeFromWFO(
                 $grid->wfo,
-                "wfo_promo",
+                "wfo_info",
             );
 
-            // If we actually have a story, pass its ID on for the template
+            // If there is WFO info, pull out the parts that should be rendered
+            // in the WFO promo block on the forecast page.
             if ($promo) {
+                $name = $promo->field_wfo->entity->name->value;
+                $code = $promo->field_wfo->entity->field_wfo_code->value;
+
+                $about = $promo->body;
+                $phone = $promo->field_phone_number_opt ?? false;
+
+                $facebook = $promo->field_facebook_url ?? false;
+                $twitter = $promo->field_twitter_url ?? false;
+                $youtube = $promo->field_youtube_url ?? false;
+
+                $social = false;
+                if ($facebook || $twitter || $youtube) {
+                    $social = [
+                        "facebook" => $facebook,
+                        "twitter" => $twitter,
+                        "youtube" => $youtube,
+                    ];
+                }
```

Comment:
Inspired by @eric-gade's comment in [#1542](https://github.com/weather-gov/weather.gov/pull/1539#discussion_r1713724374)